### PR TITLE
MBS-11263 run samples tests on CI

### PR DESCRIPTION
**Pull request by dsvoronin**
Added separate teamcity configuration, that will break PR's with sample instrumentation tests problems

### CODE REVIEW

File: ci/_environment.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-ANDROID_BUILDER_TAG=28e6bacd68
+ANDROID_BUILDER_TAG=eddf45adfa
```
Comment (eugene-krivobokov) [inline ci/_environment.sh:3]: What has been changed?

Comment (dsvoronin) [reply ci/_environment.sh:3]: https://github.com/avito-tech/avito-android/pull/813
it is changes from that pr

**Comment by sergioboi** [comment]
why do we need a separate teamcity configuration?

**Comment by dsvoronin** [comment]
> why do we need a separate teamcity configuration?

it is temporary; i'am not sure that sample project is stable enough to be a PR blocker right now, wanted to look at it for ~ a week

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**